implementation/domain/logic/jbb-system-impl/src/main/java/org/jbb/system/impl/base/properties/SystemProperties.java

```diff
@@ -17,7 +17,11 @@
 @Config.Sources({"file:${jbb.home}/jbb-system.properties"})
 public interface SystemProperties extends ModuleProperties { // NOSONAR (key names should stay)
     String STACK_TRACE_VISIBILITY_LEVEL_KEY = "stacktrace.visibility.level";
+    String SESSION_INACTIVE_INTERVAL_TIME = "session.inactive.interval.time";
```

Comment:
Missing unit of time. Is it seconds, miliseconds or what? Maybe "session.max.inactive.seconds" or "session.max.inactive.milis"?